### DISCUSSION TITLE
ParserEngine.h should include expat.h "quoted" not <angle bracketed>

### DISCUSSION THREAD

**Discussion by Vadrgh** [pocoproject/poco#4292] [Q&A]
### Summary
When using poco as a [conan2](https://conan.io/) managed dependency of a cmake project, including `Poco/SAX/SAXParser.h`  results in `Poco/XML/ParserEngine.h:22:10: error: 'expat.h' file not found`.

With _clang_ instead of _g++_ there is an additional hint:

     'expat.h' file not found with <angled> include; use "quotes" instead

After following that suggestion and patching ParserEngine.h in the conan repository / local cache, the project compiles.

#### Dev Environment
- OS: Fedora38
- Tools:
  - gcc: 13.1.1
  - clang: 16.0.4
  - cmake: 3.26.4
  - conan: 2.0.6
-  dependency specified as poco/1.12.4
#### Observations
The relevant part in HEAD of ParserEngine.h reads
```
#if defined(POCO_UNBUNDLED)
#include <expat.h>
#else
#include "Poco/XML/expat.h"
```
⇒ only for POCO_UNBUNDLED  `expat.h` is a system(-override) include.

Maybe more important: same  poco/1.12.4 worked before on Fedora36 with conan-1.58 and the toolchain belonging to that OS version,- whatever that was.

**Comment by obiltschnig** [comment]
It's not as simple to solve. Ideally, Conan should set the system include path in the correct way. A change would break things for everyone else.
